src/engine/audio/audio.main.ts

```diff
@@ -797,7 +797,7 @@ export function setLocalMediaStream(ctx: IMainThreadContext, mediaStream: MediaS
 
   if (mediaStream) {
     if (audioModule.localMediaStreamSource) {
-      throw new Error("Local media stream source already connected.");
+      console.warn("Local media stream source already connected.");
```

Comment:
I think I left that throw in there on purpose. Are we sure we want to move to just logging a warning?

Reply:
ah, yes. this was how i discovered that our errors were being swallowed but indeed forgot to revert it to a throw